index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Course Template</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+</head>
+
+<body>
+    <header>
+        <div class="header_back">
+            <div class="wrapper">
+                <div class="header_info">
+                    <ul class="info">
+                        <li><i class="pic fas fa-envelope"></i>
```

Comment:
Для блока с почтой и телефоном можно сделать так  и соотв. отталкиваться от класса info-item для внутренних отступов и положения иконки. Контент может измениться и более оптимально отстреливаться от контейнера, чем от контента.
```suggestion
                        <li class="info-item"><i class="pic fas fa-envelope"></i>
```

Reply:
Thank you for this point